Redhat8: various testcases fixes

chvalean opened the pull request:
Small fixes to test failures observed from redhat8 validation.
Results have been checked from running the code, thus not including test results.

iamshital commented:
Execution is verified.
------------
```
09/07/2018 21:13:51 : [INFO ] ~~~~~~~~~~~~~~~TEST END : VERIFY-DEPLOYMENT-PROVISION~~~~~~~~~~
09/07/2018 21:13:51 : [INFO ] CURRENT - PASS    - 1
09/07/2018 21:13:51 : [INFO ] CURRENT - FAIL    - 0
09/07/2018 21:13:51 : [INFO ] CURRENT - ABORTED - 0
09/07/2018 21:13:51 : [INFO ] Checking background cleanup jobs..
09/07/2018 21:13:51 : [INFO ] Removed 0 context files.
09/07/2018 21:13:51 : [INFO ] Cycle Finished.. TC-43912
09/07/2018 21:13:51 : [INFO ] 
Test Results Summary
ICA test run on 09/07/2018 21:06:25
ARM Image under test RedHat : RHEL : 7-RAW : latest
Total Executed TestCases        1 (       1 Pass,        0 Fail,        0 Abort)
Total Execution Time(dd:hh:mm) 0:0:7
XML file: TestConfiguration

VERIFY-DEPLOYMENT-PROVISION Execution Time: 7.38 minutes
	VERIFY-DEPLOYMENT-PROVISION : PASS  
		  FirstBoot : PASS 
		  FirstBoot : Call Trace Verification : PASS 
		  Reboot : PASS 
		  Reboot : Call Trace Verification : PASS 


Logs can be found at .\TestResults\2018-07-09-21-06-16-1500


09/07/2018 21:13:51 : [INFO ] Creating 'Azure-43912-buildlogs.zip' from '.\TestResults\2018-07-09-21-06-16-1500'
09/07/2018 21:13:51 : [INFO ] C:\LISAv2\GQMN43912\Azure-43912-buildlogs.zip created successfully.
09/07/2018 21:13:51 : [INFO ] Copied : .\report\report_TC-43912.xml --> .\report\report_TC-43912-junit.xml
09/07/2018 21:13:51 : [INFO ] Analysing results..
09/07/2018 21:13:51 : [INFO ] PASS  : 1
09/07/2018 21:13:51 : [INFO ] FAIL  : 0
09/07/2018 21:13:51 : [INFO ] ABORT : 0
Copying all files back to original working directory: C:\Jenkins-Slaves\azure-slave-2-executor-7\workspace\job-azuretest\job-cloudtest-auto@2.
Cleaning up C:\LISAv2\GQMN43912
Setting workspace back to original location: C:\Jenkins-Slaves\azure-slave-2-executor-7\workspace\job-azuretest\job-cloudtest-auto@2
09/07/2018 21:13:54 : [INFO ] LISAv2 exits with code: 0
```

chvalean committed:
Update VERIFY-VHD-PREREQUISITES.py

chvalean committed:
Update Linux-Test-Project-Tests.sh

chvalean committed:
Update VERIFY-BOOT-ERROR-WARNINGS.py

Semantics only changes, do write info if no errors are found, do write error if messages are present.

chvalean committed:
Update perf_ntttcp.sh

firewalld is enabled by default on rhel8, and that is breaking ntttcp connections.

ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
----------------------------------------------------------------------------------------------------------------------------------------
 1 NETWORK              PERF-NETWORK-TCP-THROUGHPUT-MULTICONNECTION-NTTTCP-SRIOV                          PASS                80.08
